### PR TITLE
Devin/symco 66 and 64

### PR DESCRIPTION
**Pull request by devin-w-hill**
Assignment Management page. Shows a researcher's assignments and related info. Create assignment goes to that page.
To test: create some assignments with different researchers. Have students do a few responses. All info shown should be specific to an assignment. View template responses should take you to the excel page with all template responses. Check if percent completion/student completion count is correct.
I added the completion counts and percentage for funsies, we can take out if people don't like it/think it's not useful.

### CODE REVIEW

**Comment by rosasck** [comment]
me trying to figure out what blocked means...

**Comment by devin-w-hill** [comment]
Xingjiang's ticket is blocking me :) I just need to add the button to his page though

**Review by kaigoos — CHANGES_REQUESTED** [review]
Since Xingjian's branch has been merged to main the create assignment button should be included in this PR too. Also if there are multiple templates with the same name "View All Template Responses" will cause an error. That's a mistake on my end so I will create a ticket for that bug

**Review by JerCarnahan — APPROVED** [review]
Didn't find any errors other than the ones mentioned

File: simcon_project/simcon_project/urls.py [hunk 1]
```diff
@@ -50,9 +50,11 @@
     path('researcher/view-all-responses/<uuid:pk>/', TemplateResponsesView.as_view(), name="view-all-responses"),
     path('researcher/students/<str:name>/', student_management, name="student-management"),
     path('researcher/students/', student_management, name="student-management"),
+    path('researcher/assignments/', include('users.urls.ass_man_urls'), name="assignment-management"),
```
Comment (rosasck) [inline simcon_project/simcon_project/urls.py:53]: ass_man_urls......
i know its assignment management but it made me chuckle

File: simcon_project/templates/assignment_management/view_settings_modal.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      {% endif %}
+    </div>
+    <script type="text/javascript">
+      function select_click(btn)
+      {
+        (window).open($(btn).data('form-url'), "_self");
+      }
+     </script>
+  </form>
+{% endblock %}
```
Comment (rosasck) [inline simcon_project/templates/assignment_management/view_settings_modal.html]: new line

File: simcon_project/templates/assignment_management/view_students_modal.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      {% endif %}
+    </div>
+    <script type="text/javascript">
+      function select_click(btn)
+      {
+        (window).open($(btn).data('form-url'), "_self");
+      }
+     </script>
+  </form>
+{% endblock %}
```
Comment (rosasck) [inline simcon_project/templates/assignment_management/view_students_modal.html]: newline

**Review by rosasck — APPROVED** [review]

**Review by kaigoos — APPROVED** [review]

**Pull request merged**